demo/vanilla/CHANGELOG.md

```
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.0.0-alpha.15](https://github.com/tsparticles/tsparticles/compare/v4.0.0-alpha.14...v4.0.0-alpha.15) (2026-02-03)
+
+**Note:** Version bump only for package @tsparticles/demo
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)